Configuration of configurator failed
Traceback (most recent call last):
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 171, in _load_configurators
    entry_point.load()().configure(
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 498, in configure
    self._configure(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 514, in _configure
    _initialize_components(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 426, in _initialize_components
    span_exporters, metric_exporters, log_exporters = _import_exporters(
                                                      ^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 321, in _import_exporters
    ) in _import_config_components(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 122, in _import_config_components
    ).load(),
      ^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/importlib_metadata/__init__.py", line 221, in load
    module = import_module(self.module)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py", line 22, in <module>
    from opentelemetry.exporter.otlp.proto.common.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/trace_encoder.py", line 16, in <module>
    from opentelemetry.exporter.otlp.proto.common._internal.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 30, in <module>
    from opentelemetry.proto.common.v1.common_pb2 import AnyValue as PB2AnyValue
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/proto/common/v1/common_pb2.py", line 6, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google.protobuf'
Failed to auto initialize OpenTelemetry
Traceback (most recent call last):
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/__init__.py", line 136, in initialize
    _load_configurators()
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 184, in _load_configurators
    raise exc
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 171, in _load_configurators
    entry_point.load()().configure(
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 498, in configure
    self._configure(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 514, in _configure
    _initialize_components(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 426, in _initialize_components
    span_exporters, metric_exporters, log_exporters = _import_exporters(
                                                      ^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 321, in _import_exporters
    ) in _import_config_components(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 122, in _import_config_components
    ).load(),
      ^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/importlib_metadata/__init__.py", line 221, in load
    module = import_module(self.module)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py", line 22, in <module>
    from opentelemetry.exporter.otlp.proto.common.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/trace_encoder.py", line 16, in <module>
    from opentelemetry.exporter.otlp.proto.common._internal.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 30, in <module>
    from opentelemetry.proto.common.v1.common_pb2 import AnyValue as PB2AnyValue
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/proto/common/v1/common_pb2.py", line 6, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google.protobuf'
Configuration of configurator failed
Traceback (most recent call last):
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 171, in _load_configurators
    entry_point.load()().configure(
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 498, in configure
    self._configure(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 514, in _configure
    _initialize_components(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 426, in _initialize_components
    span_exporters, metric_exporters, log_exporters = _import_exporters(
                                                      ^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 321, in _import_exporters
    ) in _import_config_components(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 122, in _import_config_components
    ).load(),
      ^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/importlib_metadata/__init__.py", line 221, in load
    module = import_module(self.module)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py", line 22, in <module>
    from opentelemetry.exporter.otlp.proto.common.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/trace_encoder.py", line 16, in <module>
    from opentelemetry.exporter.otlp.proto.common._internal.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 30, in <module>
    from opentelemetry.proto.common.v1.common_pb2 import AnyValue as PB2AnyValue
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/proto/common/v1/common_pb2.py", line 6, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google.protobuf'
Failed to auto initialize OpenTelemetry
Traceback (most recent call last):
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/__init__.py", line 136, in initialize
    _load_configurators()
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 184, in _load_configurators
    raise exc
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/instrumentation/auto_instrumentation/_load.py", line 171, in _load_configurators
    entry_point.load()().configure(
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 498, in configure
    self._configure(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 514, in _configure
    _initialize_components(**kwargs)
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 426, in _initialize_components
    span_exporters, metric_exporters, log_exporters = _import_exporters(
                                                      ^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 321, in _import_exporters
    ) in _import_config_components(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/sdk/_configuration/__init__.py", line 122, in _import_config_components
    ).load(),
      ^^^^^^
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/importlib_metadata/__init__.py", line 221, in load
    module = import_module(self.module)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py", line 22, in <module>
    from opentelemetry.exporter.otlp.proto.common.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/trace_encoder.py", line 16, in <module>
    from opentelemetry.exporter.otlp.proto.common._internal.trace_encoder import (
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 30, in <module>
    from opentelemetry.proto.common.v1.common_pb2 import AnyValue as PB2AnyValue
  File "/usr/share/opentelemetry_shell/agent.instrumentation.python/3.12/site-packages/opentelemetry/proto/common/v1/common_pb2.py", line 6, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google.protobuf'
